source/mainnet/conf.py

```diff
@@ -151,7 +151,7 @@
     # # Toc options
     "collapse_navigation": False,
     # 'sticky_navigation': True,
-    "navigation_depth": 2,
+    "navigation_depth": 3,
```

Comment:
I am not sure we want to change this _global_ menu depth setting. We should hear @dg-concordium when she is back 😊

Reply:
I discussed it with @dg-concordium already in DM. She is fine with changing the level. My initial check was that it barely affects other parts. Most other tables of content are `hidden` at level 3 meaning the navigation is barely changing on the left through this change but yes if you find anything weird we should `hide` them at level 3 if necessary.

The tutorial section is now one level deeper because we have one additional` index.rst` file that states that we have two tutorials to pick from. Without changing the global menu depth parts 1-4 of the tutorials would not be included in the navigation on the left side.

Reply:
Ok, great 😊